src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/InstallAppsTask.groovy

```diff
@@ -107,7 +108,7 @@ class InstallAppsTask extends AbstractServerTask {
         return configured
     }
 
-    private String getArchiveName(String archiveName){
+    private String checkForStripVersion(String archiveName){
```

Comment:
I changed the method name because this method is only removing the version if stripVersion is set to true. I can go ahead and change this back if you think getArchiveName() is a better choice.